pipelines/SpeciesTreeFromBusco/scripts/collate_busco_results.py

```diff
@@ -108,6 +108,8 @@ def load_sequences(infile: str) -> Tuple[Dict[str, List[SeqRecord]], int]:
 
     # Dump per-gene cDNAs:
     for g, s in per_gene.items():
+        # Filter out sequences with lengths not divisible by 3:
+        s = [x for x in s if len(x.seq) % 3 == 0]
```

Comment:
Perhaps consider using variable name than `s`  as you overwrite with the output from the list comprehension and the outer loop.

Reply:
Since s is a variable local to the loop, what would be the harm in overwriting it (which was my intention)?